framework/utilities/ecl-font-size/README.md

```diff
@@ -0,0 +1,10 @@
+# Font size utilities
```

Comment:
Please include the documentation from design wiki pages https://webgate.ec.europa.eu/CITnet/confluence/pages/viewpage.action?pageId=650250290#Typography..-Font